osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/ValidateSecurityGroupTenantTask.java

```diff
@@ -55,25 +54,19 @@ public void executeTransaction(EntityManager em) throws Exception {
         this.securityGroup = sgEmgr.findByPrimaryKey(this.securityGroup.getId());
 
         this.log.info("Validating the Security Group tenant " + this.securityGroup.getTenantName() + " exists.");
-        JCloudKeyStone keystone = new JCloudKeyStone(new Endpoint(this.securityGroup.getVirtualizationConnector()));
-
-        try {
-            Tenant tenant = keystone.getTenantById(this.securityGroup.getTenantId());
-            if (tenant == null) {
-                this.log.info("Security Group tenant " + this.securityGroup.getTenantName() + " Deleted from openstack. Marking Security Group for deletion.");
-                // Tenant was deleted, mark Security Group for deleting as well
-                OSCEntityManager.markDeleted(em, this.securityGroup, this.txBroadcastUtil);
-            } else {
-                // Sync the tenant name if needed
-                if (!tenant.getName().equals(this.securityGroup.getTenantName())) {
-                    this.log.info("Security Group tenant name updated from " + this.securityGroup.getTenantName() + " to " + tenant.getName());
-                    this.securityGroup.setTenantName(tenant.getName());
-                    OSCEntityManager.update(em, this.securityGroup, this.txBroadcastUtil);
-                }
+        Openstack4jKeystone keystone = new Openstack4jKeystone(new Endpoint(this.securityGroup.getVirtualizationConnector()));
```

Comment:
need to close keystone...please recheck all instances so we dont leave stale resources